src/DependencyInjection/AceEditorExtension.php

```diff
@@ -42,10 +58,10 @@ private function registerAceEditorParameters(array $config, ContainerBuilder $co
         $mode = 'src' . ($debug ? '' : '-min') . ($config['noconflict'] ? '-noconflict' : '');
 
         $useStimulus = $config['use_stimulus'];
-        if ($useStimulus === null) {
+        if (null === $useStimulus) {
             $bundles = $container->getParameter('kernel.bundles');
-            assert(is_array($bundles));
-            $useStimulus = in_array(StimulusBundle::class, $bundles, true) && interface_exists(AssetMapperInterface::class);
+            \assert(\is_array($bundles));
```

Comment:
native function invocation is a small performance improvement which speeds up resolving.
https://cs.symfony.com/doc/rules/function_notation/native_function_invocation.html